### DISCUSSION TITLE
Is there a straightfotrward way to use depth2image functionality with this colab?

### DISCUSSION THREAD

**Discussion by outhipped** [TheLastBen/fast-stable-diffusion#1204] [Q&A]
I'm really struggling to even try out this while running the collab., I tried installing the script but I get the following error "extensions/depthmap2mask/scripts/depthmap_for_depth2img.py". I'm not even sure if this is the Stability AI depth2image option or a earlier, thid party, implementation. 
All tutorials on youtube are about local SD installs.

Did anyone who has next to no coding skills was able to play with depth2image with thelastben colab?

Thank you.

**Comment by TheLastBen** [comment]
the depth2image extension seems to be broken in the colab after they updated it, I'll fix it shortly

**Reply by outhipped** [reply]
Thanks. I've found Spanish language tutorial using your collab when the guy rewcommends pasting the following code before the last step in the colab:
https://pastebin.com/GS7MHYV0

Maybe it can be implemented in the colab by default?
Here's the tutorial:
https://www.youtube.com/watch?v=rp3uaXtjw1M

Thanks!

**Reply by TheLastBen** [reply]
I'll add it
